### PR TITLE
[WIP] Reworked readme to be cleaner and more concise

### PR DESCRIPTION
**Pull request by ascott18**
Removed large parts that largely existed to try and justify the existence of Coalesce and the choices made - lets focus instead on just what it does. We can put the story of Coalesce's history somewhere else.

Related to the work for #85

Looking for feedback - WIP for now so it doesn't get merged for now.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,75 +1,38 @@
...
-Projects that used to take us three weeks now take one week. 
+Coalesce is a framework for rapid-development of ASP.NET Core web applications. It works from the Entity Framework Core data model that you design, automating the creation of the glue - DTOs, API Controllers, and TypeScript - that sit between your data and the UI of your application. 
 
-## Learn More
-You can find a sample app with documentation here: http://coalesce.intellitect.com/
+## Fundamentals
 
-## Get Started Fast
-We recommend using our starter project at at: https://github.com/IntelliTect/Coalesce.Starter
+* **Code Generation** Write your data model, and we'll build the DTOs, API controllers, and TypeScript that are needed to start building a front-end UI right away. A full suite of CRUD endpoints are created for you, with inner-workings that are overridable to great degrees.
```
Comment (Keboo) [inline README.md]: TypeScript is singular while the other items are plural. Perhaps "TypeScript files"?

Comment (Keboo) [inline README.md]: You can omit "to great degrees"

Comment (ascott18) [reply README.md]: Thanks - I have a tendency to add fluff words all over the place, but I wanted to keep them out of this as much as possible.

Comment (ascott18) [reply README.md]: 👍

Comment (WissingWriter) [reply README.md]: "A full suite of CRUD endpoints are created for you, with inner-workings that are overridable to great degrees."
Is a bit choppy. Does "A full suite of CRUD endpoints with overridable inner-workings are available for your use" make sense?

Comment (WissingWriter) [reply README.md]: There's an extra at before the link under getting started fast.

Comment (WissingWriter) [reply README.md]: I'd add a : after "for" in the first section - "code generation framework for:"

**Pull request merged**